.vortex/cli/src/Command/UpdateCommand.php

```
@@ -0,0 +1,98 @@
+  /**
+   * Resolve the repository URI to download.
+   *
+   * @param mixed $to
+   *   The target version, if any.
+   * @param mixed $uri
+   *   The explicit repository URI, if any.
+   *
+   * @return string|null
+   *   The URI to download, or NULL to leave resolution at its default.
+   */
+  public function targetUri(mixed $to, mixed $uri): ?string {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $to. Configured minimum length is 3. (undefined)

(ShortVariable)